templates/data-providers/index.jsx

```diff
@@ -25,32 +15,27 @@ const DataProviderPageTemplate = React.forwardRef(
 
     return (
       <Card className={styles.layout}>
-        {showSuccessMessage && <SuccessMessage oaiPmhUrl={oaiPmhUrl} />}
-        {!showSuccessMessage && (
-          <>
-            <Card.Title>Register new data provider</Card.Title>
-            <form ref={ref} className={styles.form} onSubmit={handleSubmit}>
-              <TextField
-                id="data-provider-url"
-                type="url"
-                name="dataProviderUrl"
-                label="Data provider URL"
-                helper={helperMessage}
-                placeholder="For example: oro.open.ac.uk"
-                value={url}
-                onChange={onUrlChange}
-                variant={variant}
-                statusIcon
-                required
-              />
-              <div className={styles.submitSection}>
-                <Button variant="contained" disabled={!isFormValid}>
-                  Continue
-                </Button>
-              </div>
-            </form>
-          </>
-        )}
+        <Card.Title>Register new data provider</Card.Title>
+        <form ref={ref} className={styles.form} onSubmit={handleSubmit}>
+          <TextField
+            id="data-provider-url"
+            type="url"
+            name="dataProviderUrl"
+            label="Data provider URL"
+            helper={helperMessage}
+            placeholder="For example: oro.open.ac.uk"
+            value={url}
+            onChange={onUrlChange}
+            variant={variant}
+            statusIcon
+            required
+          />
+          <div className={styles.submitSection}>
+            <Button variant="contained" disabled={!isFormValid}>
```

Comment:
Shouldn't be button visible only when new data provider is created? Also maybe the better naming would be `Add another data provider` or something like that.

Reply:
It would be a very long caption for the button. The button should be simple and describe in one-two words.

_Done_ should be fine, in my opinion. The message clearly says, that we 'noted the URL already' (updated in #52)